codice/ddf
# DDF-2040/DDF-2041: Adding security auditing for web context policy co…

#### What does this PR do?
Adds security auditing when the web context policy manager is changed and when an admin signs in or out
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@roelens8 @ryeats @jckilmer @coyotesqrl @bcwaters 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@stustison 
@pklinef 
#### How should this be tested?
Verify the security.log contains changes to the web context policy manager and admin logging in and out. Make sure to also verify idp logging in as well
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Checklist:

…nfiguration and admin sign in/out
